utils/kubernetes/expose/error.go

```diff
@@ -0,0 +1,104 @@
+package expose
+
+import (
+	"fmt"
+
+	"github.com/layer5io/meshkit/errors"
+)
+
+var (
+	// ErrGettingResourceCode is generated when there is an error getting the kubernetes resource
+	ErrGettingResourceCode = "meshkit_test_code"
+
+	// ErrTraverserCode is a collection of errors generated during traversing the resources
+	ErrTraverserCode = "meshkit_test_code"
```

Comment:
We're about ready to start defining error codes... will you propose an approach so that we can review and move forward with their definitions?

Reply:
We have defined some of the test codes in the meshkit, will need some guidelines from @kumarabd regarding the same.

Reply:
Ok... later.